projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -23,6 +25,20 @@ import { TableMode } from '../table-api';
           (valueChange)="this.onSearchChange($event)"
         ></ht-search-box>
 
+        <!-- Selects -->
+        <ht-select
```

Comment:
Shouldn't the select dropdown be a multi select dropdown? For example: Show me all the APIs which lies in Service A or Service B or Service C

Reply:
This is actually good feedback. I think multi select makes sense. That said, let me get the single select version out and demoed. I'll make the switch to IN filtering once everyone is onboard with the new changes.

Reply:
yes. that makes sense

Reply:
> This is actually good feedback

What are you saying?!